app/krs.js

```diff
@@ -159,7 +159,7 @@ exports.provisionKey = co(function *(req) {
 
   yield masterKey.update({ $inc: { keyCount: 1 } });
 
-  if (!req.body.disableKRSEmail) {
+  if (!req.body.disableKRSEmail && !process.config.disableAllKRSEmail) {
```

Comment:
If the config option specifically says `all`, I think this check should be in sendMailQ, so that other emails that we send can be blocked.